GeneratorAnalysis/Scripts/FlatTreeAnalyzer.cxx

```diff
@@ -46,7 +46,8 @@ void FlatTreeAnalyzer::Loop() {
     //----------------------------------------//	
 
     // Output file
-    TString Directory = "/pnfs/sbnd/persistent/users/" + (TString)UserName + "/HighSamples/";
+    TString Directory = "/pnfs/sbnd/persistent/users/theobal1/HighSamples/";
+    //TString Directory = "/pnfs/sbnd/persistent/users/" + (TString)UserName + "/HighSamples/";
```

Comment:
I think for merging, we want to keep the `+(TString)UserName+` part to make it easy for other people to use